### ISSUE TITLE
Running `website-scraper` Without a `package.json` in the Working Directory Triggers Error

### ISSUE THREAD

**Issue by manuth** [website-scraper/node-website-scraper#512]
### Configuration
**version:** `5.3.0`
**options:** _options don't have an influence on this issue_

### Description

When running `website-scraper` with the current working directory set to a directory which does not contain a `package.json` file triggers an error.

**Expected behavior:** The working directory should not have any side-effect of this sort

**Actual behavior:** An error is thrown:
```
ENOENT: no such file or directory, open './package.json'.
```

### Additional Information

This error is caused by this piece of code:
https://github.com/website-scraper/node-website-scraper/blob/de75674c78a6c84783c3b017c0518b427f28d769/lib/config/defaults.js#L2-L3

As seen in this piece of code, this statement relies on a `package.json` file to exist in the current working directory.
A workaround is sadly not possible at time of writing.

**Comment by s0ph1e** [comment]
Hi @manuth 👋 

Thank you for reporting the problem
Could you please share the steps to reproduce this issue? How do you run your application?

**Comment by manuth** [comment]
Hey @s0ph1e 
Thanks for the rapid answer :smile:

Sorry for the inconvenience, I'll describe the steps for the reproduction real quick.

1. Create an empty directory  
    ```bash
    mkdir scrape-test && cd scrape-test
    ```
2. Create `package.json` as described below
3. Run `npm install`
4. Install dependencies
    ```bash
    npm i website-scraper && npm i -D @types/node @types/website-scraper
    ```
5. Create `index.js` file as described below
6. Run the script using `node ./index.js`
7. Take note that everything works as expected
8. `cd` into a directory which does not contain a `package.json` (for example `cd ..`)
9. Run the script `node ./scrape-test/index.js`
10. Take note that an running the script triggers an error now

### Assets
**`package.json`:**
```json
{
  "type": "module",
  "dependencies": {
    "website-scraper": "^5.3.0"
  },
  "devDependencies": {
    "@types/node": "^18.7.23",
    "@types/website-scraper": "^1.2.6"
  }
}
```

**`index.js`**
```js
import { join } from "path";
import { fileURLToPath } from "url";
import websiteScraper from "website-scraper";

(
    async () =>
    {
        await websiteScraper(
            {
                directory: join(fileURLToPath(new URL(".", import.meta.url)), "output"),
                urls: [
                    "https://nodejs.org"
                ]
            });
    })();
```

**Comment by s0ph1e** [comment]
Thanks @manuth 👍

I'll try to reproduce and fix it during next 1-2 weeks

If anyone wants to contribute - PRs are welcome

**Comment by s0ph1e** [comment]
The bug was fixed and released in [v5.3.1](https://github.com/website-scraper/node-website-scraper/releases/tag/v5.3.1)

**Comment by manuth** [comment]
Awesome! 😄 Thanks for the rapid fix
